guideline-03.md

```diff
@@ -1,5 +1,5 @@
 <h4>3. A stable version of a plugin must be available from its WordPress Plugin Directory page.</h4>
 
-The only version of the plugin that WordPress.org distributes is the one in the directory. Though people may develop their code somewhere else, users will be downloading from the directory, not the development environment.
+The only version of the plugin that WordPress.org distributes is what is hosted in the directory. Authors may develop their code where they wish, but users will only download the completed product from the WordPress.org directory.
 
-Distributing code via alternate methods, while not keeping the code hosted here up to date, may result in a plugin being removed.
+Distributing code via alternate methods, while not keeping the code hosted on WordPress.org up to date, is not permitted.
```

Comment:
Does this mean they can't offer a newer version as a download from their own site?  For example, authors might want to do a slow release via their own site before opening the floodgates when its uploaded to wordpress.org

Reply:
As long as you're _planning_ to keep the code updated here, that's fine. We're talking about people who put a known-bad version on .org and then the GOOD version up on their site, with no intention to update here. They just want the SEO juice.